app/controllers/sign_up/completions_controller.rb

```diff
@@ -55,16 +55,17 @@ def completions_presenter
       )
     end
 
-    def ial2?
-      sp_session[:ial2]
+    def identity_proofing_required?
+      resolved_authn_context_result.identity_proofing?
```

Comment:
probs a thought for a separate PR:

this is making me think `resolved_authn_context_result` might not be the clearest name? because otherwise we could essentially make this is a `delegate_to` kind of thing, renaming the property for clarity makes it seem like we have two names for the same thing

 Like this is what the RP requested, so maybe it's like `authentication_request`?

Reply:
Yeah, I didn't love "AuthnContextResolver" and it has been turtles all the way down. Open to other suggestions.

Reply:
- `requested_authentication` would let us keep the same property names so `requested_authentication.identity_proofing?` makes it clear it was what was requested?
- `auth_request.identity_proofing_required?` might be another?